go/private/BUILD.sdk.bazel

```diff
@@ -1,6 +1,6 @@
-load("@io_bazel_rules_go//go/private/rules:binary.bzl", "go_tool_binary")
-load("@io_bazel_rules_go//go/private/rules:sdk.bzl", "package_list")
-load("@io_bazel_rules_go//go:def.bzl", "declare_toolchains", "go_sdk")
+load("@{rules_go_repo_name}//go/private/rules:binary.bzl", "go_tool_binary")
```

Comment:
How does this work? Isn't this invalid syntax for a label? Can we add some comments describing why this would work?

Reply:
There is no magic, it's just a placeholder filled in in `sdk.bzl`, just like the other variables in this BUILD file template. Should I still add a comment?

Reply:
The more I stare at this the less I care about comments. Sorry to bother.